man/.gitignore

```diff
@@ -0,0 +1 @@
+derive_var_trtedtm.Rd
```

Comment:
Hey Jack, I don't think this should be here.  You can use git revert to get the .gitgnore back to its original state.

Reply:
Hi @bms63 oh no I have no idea why that changed. Looking into how to revert this now!

Reply:
You can manually fix this if git revert is not working.

Reply:
That occurred to me to do. 

A bit confused as to what exactly a .gitignore file is though and where it came from! Is it fair to assume that fixing it would just be deleting the `.gitignore` file from /admiral/man? I had a look at `devel` and there isn't one there so somehow I think I created one and put the name of that file in there.